src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -32,9 +70,15 @@ class CDCL(val clauses: MutableList<MutableList<Int>>, initNumber: Int = 0) {
         }
     }
 
+    data class VarState(
+        var status: VarStatus,
+        var clause: Clause?,
+        var level: Int,
+    )
+
     // get status of literal
-    private fun getStatus(lit: Int): VarStatus {
-        if (vars[litIndex(lit)].status == VarStatus.UNDEFINED) return VarStatus.UNDEFINED
+    fun getStatus(lit: Int): VarStatus {
```

Comment:
`value` better describes this, than a `status`. "Status" can mean much more than just an assignment value...
So, consider `getValue(lit: Lit) -> lbool` or even just `value(lit: Lit) -> lbool`.